consensus/tip_selection/exit_probability_randomizer/tests/test_exit_probability_randomizer.c

```diff
@@ -224,8 +224,8 @@ void test_single_tx_tangle_walker(void) {
 
 void test_single_tx_tangle_map(void) {
   ep_prob_map_randomizer_t ep_randomizer;
```

Comment:
This one doesn't need to be initialised 🤔 ?

Reply:
seems the test_cw_gen_topology() takes const variable, and the test_single_tx_tangle_base() takes non-const variable, and const. values should be initialized while they are declared.